html/themes/custom/common_design_subtheme/templates/cd/cd-footer/cd-footer.html.twig

```diff
@@ -18,7 +18,7 @@
       {% include '@common_design_subtheme/cd/cd-footer/cd-copyright.html.twig' %}
     {% endblock %}
 
-    {% if logged_in %}
+    {% if ('editor' in user.getroles) or ('webmaster' in user.getroles) or ('administrator' in user.getroles) or (user.id == '1')%}
```

Comment:
For info, at some point, the module that handles a few things related to users will be migrated. We can add a permission for the admin menu there and remove the check on the roles.